config/jobs/kubernetes/sig-cloud-provider/gcp/gcp-gce.yaml

```diff
@@ -13,7 +13,7 @@ presets:
     preset-use-new-registry: "true"
   env:
   - name: KUBE_TEST_REPO_LIST
-    value: https://raw.githubusercontent.com/kubernetes/test-infra/master/config/jobs/kubernetes/sig-cloud-provider/gcp/registry/default.yaml
+    value: https://raw.githubusercontent.com/kubernetes/test-infra/master/config/jobs/kubernetes/sig-cloud-provider/gcp/registry/default.txt
```

Comment:
Rather than giving this an incorrect file extension, we can direct Prow to ignore it using a `.prowignore` file (which behaves like a `.gitignore` file).

Reply:
oh thats perfect... I wasnt sure if there was one... Ill add it in now :pray:

Reply:
done